dub.json

```diff
@@ -12,11 +12,11 @@
     "StdLoggerDisableWarning"
   ],
   "dependencies" : {
-    "libdparse": ">=0.17.0 <0.18.0",
+    "libdparse": ">=0.18.0 <0.19.0",
```

Comment:
0.18.0 is only needed for new syntax, but not needed for any new analyzers. As older versions still work and may be used by other tools you should keep this lowered at 0.17.0 until there are new checks actually utilizing it.